components/Layout.tsx

```diff
@@ -12,11 +13,13 @@ const style = {
 const Layout = ({ children }: Props) => {
     return (
         <WalletProvider>
-            <Head>
-                <title>GammaSwap</title>
-                <link rel="icon" href="/favicon.ico" />
-            </Head>
-            <main className={style.wrapper}>{children}</main>
+            <BasicContractProvider>
```

Comment:
Definitely. Mainly to prevent [callback hell](https://www.reddit.com/r/javascript/comments/5rjrcy/why_is_twospace_tab_indent_becoming_the_standard/). When we get his changes merged, it'll adjust automatically.